roles/solrcloud/tasks/install.yml

```diff
@@ -1,54 +1,38 @@
 ---
-# installation instructions taken from: https://lucene.apache.org/solr/guide/7_2/taking-solr-to-production.html#taking-solr-to-production
-
-- name: solrcloud | create solr dirs
-  ansible.builtin.file:
-    path: '{{ item }}'
-    state: directory
-    owner: '{{ solr_user }}'
-    group: '{{ solr_group }}'
-    mode: 0755
-  loop:
-    - '{{ solr_log_dir }}'
-    - '{{ solr_data_dir }}'
```

Comment:
Is this the only place we used these vars? If so, then we don't need them and can remove them from defaults and group_vars.